pages/productListingPage/productListingPage.css

```diff
@@ -0,0 +1,135 @@
+@import "/styles.css";
+
+:root {
+}
+
+body {
+}
+
+.nav {
+}
+
+.nav__logo {
+}
+
+.nav__search {
+}
+
+.nav__actions {
+}
+
+.nav__login-btn {
+    padding-left: 2.8rem;
+    padding-right: 2.8rem;
+}
+
+/* .nav__wish-lst-component {
```

Comment:
remove commented code if it's not kept for future use

Reply:
Done.